### PR TITLE
Initial hugo setup and content

### PR DESCRIPTION
**Pull request by davidhollis**
Creates a hugo site with an fodt template and two pages (a homepage and a donate page).

Screenshot (chrome, macOS):

Screen Shot 2020-04-25 at 10 19 51 AM

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+baseURL = "https://friendsofdt.org/"
```
Comment (c1tadel) [inline config.toml:1]: Not sure how this changes with multiple deployment targets

File: content/_index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (c1tadel) [inline content/_index.md:1]: TODO sync all content updates, include changes pending for this weekend

File: content/donate.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Donations can be made on-line via the <a href="https://development.gatech.edu" target="blank">Georgia Tech Office of Development's website</a>.
+
+1. Click on the **Give Now** link at the top.
+2. Enter the amount of your gift in the **Donation amount** box.
+3. Enter the amount _again_ in the **Other designation** box.
+4. In the **If other please specify** box, enter `Gay Dull DramaTech Scholarship 997-00-0150`.
+5. Once you receive a confirmation email from Georgia Tech, please complete [this form][progress-form] so that we can keep track of our progress.
+
+If you run into any problems or have any questions about this process, please reach out to [the treasurer](mailto:treasurer@friendsofdt.org).
```
Comment (c1tadel) [inline content/donate.md:23]: Maybe there's a plugin to nerf these?

File: themes/fodt/layouts/partials/footer.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<footer class="content">
+  <p>
+    &copy; 2020 Friends of DramaTech<br>
+    The content of this site is made available under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons BY-NC-SA 4.0 license</a>.<br>
+    Cover photo by <a href="https://unsplash.com/@manosgk">Manos Gkikas</a>.
```
Comment (c1tadel) [inline themes/fodt/layouts/partials/footer.html:5]: +1

**Review by c1tadel — APPROVED** [review]

File: config.toml (hunk 1 above)
Comment (davidhollis) [reply config.toml:1]: It can be overridden on the command line, so we can make staging deploys point to the right place.

**Pull request merged**